share/governance.md

```diff
@@ -0,0 +1,111 @@
+
+Membership of the PMC is by election. [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) voting is held once a year. A non-running election official is appointed by the PMC and runs the election. There are 5 members of the PMC. Terms are 1 year.
+
+The PMC doesn't have specific roles or a chairperson.
+
+
+
+Specialized Roles
+-----------------
+
+
+There are three officer roles in VoxPupuli. They can all be held by the same person (though this isn't ideal), as long as the work is getting done. They can have any other role in the organization, including being on the PMC. The PMC is responsible for requesting volunteers and appointing a volunteer to the role. Appointments last until either the officer steps down, or the unlikely and unfortunate situation where the PMC removes the officer.. The PMC can add more officer roles as it sees fit by modifying this document.
+
+1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating
+
+2) Security Officer: This is the point of contact for external or internal security issues, this person has a published gpg key, and will be the main point of contact for CVE numbers and such
+
+3) Communications Officer: This is the main point of contact for external and internal publicity and marketing efforts and requests
+
+
+Lazy Consensus
+--------------
+
+Lazy consensus is a very important concept within the project. It is this process that allows a large group of people to efficiently reach consensus, as someone with no objections to a proposal need not spend time stating their position, and others need not spend time reading such statements.
+
+For lazy consensus to be effective, it is necessary to allow at least 72 hours before assuming that there are no objections to the proposal. This requirement ensures that everyone is given enough time to read, digest and respond to the proposal. This time period is chosen so as to be as inclusive as possible of all participants, regardless of their location and time commitments.
+
+We do require one affirmative vote as part of the Lazy consensus model.
+
+Vox Pupuli has a public mailing list. This list is low-traffic and is for discussion of anything Vox Pupuli related. Controversial patches, nominations for promotion, policy shifts, and anything else can be brought to this list. All contributors and PMC members are expected to be on this list. Release announcements will go out to this list, but will have ``[Release]`` in the subject line so they can be easily filtered.
+
+
+
+
+PMC Removal
+-----------
+
+A member of the PMC can be removed before the end of their 1 year term by a supermajority (2/3) vote of the PMC. This can happen for a number of reasons, but the expected reason is following a Code of Conduct violation incident. A member of the PMC can step down at any time.
```

Comment:
What _specifically_ triggers the removal vote? As it stands, there is no bar to clear before having the vote, just a suggestion of what might do it. 

Are the other members of the PMC required to bring evidence of a CoC violation? If there is a non-CoC problem (unresponsive PMC member, irresponsible actions, etc) that might warrant removal, limiting the reason for a removal vote to CoC violations would fall short and fail to protect the community in that instance.

However, I feel like we should make clear what the requirements are for impeachment, even if the answer is "there are no requirements."